vpn-router-web/src/main/java/vpnrouter/web/eventhandler/ClientDetectionFoundEventHandler.java

```diff
@@ -0,0 +1,48 @@
+package vpnrouter.web.eventhandler;
+
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.UIDetachedException;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.component.progressbar.ProgressBar;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import vpnrouter.api.event.EventSubscriber;
+import vpnrouter.api.event.EventSubscriberRegistry;
+import vpnrouter.api.event.concrete.client.ClientDetectionClientsFoundEvent;
+
+@Component
+@RequiredArgsConstructor
+public class ClientDetectionFoundEventHandler {
```

Comment:
Also, you mess with names I don't know why. The name of this class is literally incorrect.